web/public/locales/no-NB.json

```diff
@@ -1,270 +1,301 @@
+    "feedback-question": "Kan jeg gi tilbakemeldinger?",
+    "groupName": "Om oss",
+    "whoAreYou-answer": "Tjenesten er utviklet og vedlikeholdt av <a href=\"https://electricitymaps.com/\" target=\"_blank\">Electricity Maps</a>, som er et dansk oppstartselskap. Selskapets mål er å organisere elektrisitetsdata for å bidra til overgangen til et utslippsfritt energisystem.",
+    "whoAreYou-question": "Hvem har laget dette?",
+    "workTogether-answer": "Vi er på jakt etter muligheter for å samarbeide. <a href=\"mailto:app@electricitymaps.com\">Send oss en e-post!</a>",
+    "workTogether-question": "Kan vi samarbeide?"
   },
-  "onboarding-modal": {
-    "view1": {
-      "header": "Kartlegger klimaeffekten av el-energi",
-      "text": "Her kan du utforske klimaeffekten av elektrisitetsproduksjon globalt. <br/><br/>La oss ta deg rundt på en tur gjennom kartet ved hjelp av noen få enkle steg!"
-    },
-    "view2": {
-      "header": "Hvor mye CO₂ blir sluppet ut for å produsere din elektrisitet, vist i sanntid",
-      "text": "Vi fargelegger områdene i verden etter karbonintensitet (som inkluderer alle drivhusgasser) for elektrisitet forbrukt der. Jo grønnere fargen er, jo mer klimavennlig er elektrisiteten som forbrukes der."
-    },
-    "view3": {
-      "header": "Lær hvor din elektrisitet kommer fra",
-      "text": "Piler på kartet viser overføring av elektrisitet mellom ulike områder. Klikk på et område for å få mer informasjon om hvike energikilder som forbrukes i området."
-    },
-    "view7": {
-      "header": "Visning i sanntid av forholdene for vind- og solenergi",
-      "text": "Vind- og solenergiproduksjon er væravhengig. Klikk på sol- og vind-knappene for å få oversikt over forholdene for solskinn og vind i verden akkurat nå."
-    },
-    "view4": {
-      "header": "Bytt mellom forskjellige visninger",
-      "text": "Bytt mellom elektrisitetsproduksjon og -forbruk. Eller bytt mellom inndeling i land eller markedssoner for å få innsikt i endringene i energimarkedet slik at du kan gjøre informerte valg for en bærekraftig fremtid."
-    },
-    "view5": {
-      "header": "Bytt mellom visning av utslipp og elektrisitet",
-      "text": "Bytt mellom to alternative visninger som viser en el-markedssone informasjon om enten elektrisitet eller utslipp. Verdiene som vises representerer totalen for den valgte perioden, og informasjonen er sammenlignbar med tallene for produksjon eller forbruk."
-    },
-    "view6": {
-      "header": "Utforsk historiske data",
-      "text": "Nede til venstre, kan du se både sanntids og historiske utlipps- og elektrisitetdata for verden. Du kan endre på de aggregerte dataene ved bruke bryterne for å endre på tidslinja slik at du kan se utvikling over tid."
-    }
+  "aggregateButtons": {
+    "country": "land",
+    "zone": "prisområde"
```

Comment:
We should not use price area as that may not always be the case. This can be but are not limited to administrative areas, island boundaries and bidding zones.

So we need to make this wording more general.

Reply:
The sub-country regions are indeed "price areas/bidding zones" in my country. 
May be generalized to "**region**" for countries where these are not "price areas/bidding zones".

Reply:
Zone should be translated to Sone though right?

I think that would be better than region.

Reply:
**Region** is understood as sub-country division, and perhaps clearer. 
**Sone** have a less clear meaning.

Reply:
> **Region** is understood as sub-country division, and perhaps clearer. **Sone** have a less clear meaning.

It does not need to be a subarea of a country though, all existing zones are but we expect that in the coming years there will be multi country hubs for electricity production in the north and Baltic Sea for example. There could also be a cross border area (Ireland could be such a option for example).

Reply:
Ok, maybe "**Sone**" is lowest common denominator for this geographical division.

It would have been great if it had been possible to standardise this sub-division to "prize zone" / "bidding zone" with same meaning across countries.

Reply:
Yes, but currently it is not standardised across the world, and even if it was at the current state of our data sources it would not be possible to use it everywhere anyhow.